### PR TITLE
4.13 installation doc updates from IBM P

### PR DESCRIPTION
**Pull request by alishaIBM**
**Version(s) :** 4.13+

**Issue :** https://issues.redhat.com/browse/MULTIARCH-3397

**Link to docs preview :** 

- Hardware deprecation note : https://56183--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_power/installing-ibm-power.html#minimum-ibm-power-system-requirements_installing-ibm-power

**QE review :** @manojnkumar 
- [x] QE has approved this change.

**Additional information:**
https://docs.google.com/document/d/1baIzDJlqwXM7yXCTnhDe7jSciUMVyL3UgCNsMaMJJQM/edit#heading=h.iu41j0323lc2

### CODE REVIEW

**Comment by alishaIBM** [comment]
@manojnkumar jfyi : 
FIPS update in install documents is covered in a separate PR : https://github.com/openshift/openshift-docs/pull/57725
Update to RHEL 9.2 in install documents is also covered in a separate PR : https://github.com/openshift/openshift-docs/pull/57055

**Review by jeana-redhat — APPROVED** [review]
LGTM

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: modules/minimum-ibm-power-system-requirements.adoc [hunk 1]
```diff
@@ -13,7 +13,7 @@ You can install {product-title} version {product-version} on the following IBM h
 
 [NOTE]
 ====
-Support for {op-system} functionality for all IBM POWER8 models, IBM POWER9 AC922, IBM POWER9 IC922, and IBM POWER9 LC922 is deprecated. These hardware models remain fully supported in {product-title} 4.12. However, Red Hat recommends that you use later hardware models.
+Support for {op-system} functionality for all IBM Power8 models, IBM Power AC922, IBM Power IC922, and IBM Power LC922 is deprecated. These hardware models are no longer supported in {product-title} {product-version}. Red Hat recommends that you use later hardware models.
```
Comment (bergerhoffer) [inline modules/minimum-ibm-power-system-requirements.adoc:16]: @alishaIBM doesn't this note kind of contradict itself now? The first sentence says that support is deprecated, and then the second says that they are not supported at all.

Should the first two sentences be combined to say that these are no longer supported?

**Review by bergerhoffer — COMMENTED** [review]
@alishaIBM Question for you on the statement

File: modules/minimum-ibm-power-system-requirements.adoc (hunk 1 above)
Comment (alishaIBM) [reply modules/minimum-ibm-power-system-requirements.adoc]: @bergerhoffer I have updated the above sentence. Request you to check. Thanks.

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Comment by mburke5678** [comment]
@alishaIBM Once we clarify [the _recommend_ issue](https://github.com/openshift/openshift-docs/pull/56183#discussion_r1168582916), Looks Good To Merge,

**Pull request merged**